extensions/adobe/experience/experienceevent-edgeregion.schema.json

```diff
@@ -0,0 +1,192 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/experienceevent-edgeregion",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Data capture region for Experience Events",
+  "description": "Edge region location and timestamp where experience event data is captured",
+  "type": "object",
+  "meta:status": "experimental",
+  "meta:extensible": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "",
+  "definitions": {
+    "experienceevent-edgeregion": {
+      "properties": {
+        "https://ns.adobe.com/experience/userActivityRegion": {
+          "title": "User activity region",
+          "description": "Region info where event is captured.",
+          "type": "object",
+          "properties": {
+            "https://ns.adobe.com/experience/dataCenterLocation": {
```

Comment:
@ankushkd you can avoid unnecessary nesting **_experience**.dataCentreLocation by the XED converter, by just naming the property by "dataCentreLocation" instead of the URI. Check if you want to make this change?